src/components/Helper/IconHelper.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+import Markdown from 'decentraland-gatsby/dist/components/Text/Markdown'
+import TokenList from 'decentraland-gatsby/dist/utils/dom/TokenList'
+import { Popup, PopupProps } from 'decentraland-ui/dist/components/Popup/Popup'
+
+type Props = Pick<PopupProps, 'position'> & {
+  text: string
+  containerClassName?: string
+  icon: string
+}
+
+function IconHelper({ position, text, containerClassName, icon }: Props) {
+  const useContainerClassName = containerClassName && containerClassName.length > 0
```

Comment:
I'd change this to `hasContainerClassName` as the `use*` prefix is used for hooks 🤔